ca_gazebo/launch/prueba.launch

```diff
@@ -0,0 +1,29 @@
+<launch>
+	<arg name="env" default="prueba"/>
```

Comment:
Don't use names in Spanish.

Reply:
This is an unnecessary file. Deleted it.